app/controllers/hub/security_controller.rb

```diff
@@ -17,7 +17,7 @@ def show
 
     def duplicate_bank_client_ids
       return [] unless @client.intake.bank_account.present?
-      Intake.where(id: @client.intake.bank_account.duplicates.pluck(:intake_id)).pluck(:client_id)
+      @client.intake.bank_account.duplicates.map { |ba| ba.intake.client_id }.uniq
```

Comment:
it's definitely not more efficient, probably severely inefficient for any cases where the array of `duplicates` has more than a couple things in it (not sure what the dupiest bank account we have is)

I was just trying as a quick fix to normalize between `BankAccount` objects which have an `intake_id` and `Archived::BankAccount2021` objects which have an `archived_intakes_2021_id`. If we just fetch the `intake` (which is configured correctly on both models to use the right keys/tables) it will get the data we need, though it's less efficient that we need to `map` over all the bank accounts and fetch intakes one-by-one to do so.

I could've also probably tried to run two big queries that tried to fetch all the dupes from the `bank_accounts` table and all the dupes from the `archived_intakes_2021` table but I didn't want to have to mess with the definition of #duplicates at all so I left it in this odd state